src/ignitus-Internals/index.js

```diff
@@ -24,8 +24,9 @@ export class ErrorBoundary extends React.Component {
     if (errorInfo) {
       return (
         <div className="container">
-          <h2>Something went wrong.</h2>
-          <details style={{ whiteSpace: 'pre-wrap' }}>
+          <h2 style={{ textAlign: 'center', marginTop: '40px', color: '#000066' }}>Unable to load the contributors!</h2>
+          <h5 style={{ textAlign: 'center', marginTop: '20px' }}>This can happen if you are not connected to internet or if and underlying<br/>system or component is not available</h5>
+          <details style={{ whiteSpace: 'pre-wrap', textAlign: 'center', fontSize: '15px', marginBottom: '20px', paddingLeft: '380px', paddingRight: '380px' }}>
```

Comment:
Please change the padding on both the sides to `210px` to be on the safe side if the error text lines are larger. That way text will look nicer.